global.yaml

```
@@ -464,6 +464,15 @@ global:
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
 
+  # Configures DNS certificates provisioned through Chiron linked into Pilot.
+  # Te values in this file are all hard-coded; please ensure the namespaces
```

```suggestion
  # The DNS names are all hard-coded; please ensure the namespaces
```
Point to which values need to be kept in sync.
And in the comments for those values, add comments to point back to here.

---

Done. For the config in istio-control/istio-discovery/templates/configmap.yaml that uses this field, the config already points back to here as: .Values.global.certificates.